src/routes/github/instance/github-instance-get.ts

```diff
@@ -0,0 +1,90 @@
+import { NextFunction, Request, Response } from "express";
```

Comment:
What have we here!!?!?

This is the `/create-branch-options-get` file moved and slighlty modified to route` github/instance-get`

**Opinions encouraged!!!!!**

This moves the route under GitHub 💚 
It is instance agnostic, so it doesnt car if we know the uuid or not its a know nothing at this stage. 💚 
Its re-usable... mostly instead of redirecting to create-branch it takes a redirect query param 💚 

It does feel a little messy in its placement in the router specifically that its part of the subrouter that gets thrown on here `GithubRouter.use(`/:uuid(${UUID_REGEX})?`, subRouter)`  but maybe thats ok???? 🔴